Makefile

```diff
@@ -46,9 +46,11 @@ vendor: .cargo/config
 target/release/$(BIN) target/release/lib$(BIN).so target/include/$(BIN).h target/pkgconfig/$(BIN).pc.stub:
 	if [ -d vendor ]; \
 	then \
-		cargo build --release --frozen; \
+	    cargbo build --lib --release; \
```

Comment:
`cargbo` should be `cargo`

Reply:
Saw that, will be fixing it with the next commit I'm about to make.

Reply:
Cool, thanks

Reply:
It is fixed now. Had to change it to:

```sh
cargo rustc --lib --release -- --crate-type=lib;
cargo rustc --lib --release -- --crate-type=dylib;
cargo build --bin distinst --release;
```

Noticed that unless each lib crate type was generated on independent cargo runs, the build would fail.

Reply:
Ah, I see.